community/src/crawler.py

```
@@ -253,57 +272,53 @@ def crawl_stock_comments(self, stock_symbol):
-        
         logger.info("⏳ 페이지 안정화 대기...")
+        
         self.page.evaluate("window.scrollBy(0, 400);")
+        
         time.sleep(random.uniform(3, 7))
         
         try:
             self.page.wait_for_load_state("networkidle", timeout=15000)
             logger.info("✅ 네트워크 안정화 완료")
+        
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)